### ISSUE TITLE
Double XML Header when exporting SVG

### ISSUE THREAD

**Issue by rplc** [Liquid-JS/qr-code-styling#1]
Hello,

I'm using the current library version. When I download a qr code as svg, the xml header is added twice, resulting in an invalid svg.

Reproduction
```
<!doctype html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <title>QR Code Styling</title>
  </head>
  <body>
    <div id="canvas"></div>
    <button type="button" id="dl">Download</button>
    <script type="module">
    import { QRCodeStyling, browserUtils } from "https://unpkg.com/@liquid-js/qr-code-styling/lib/qr-code-styling.js";

    const qrCode = new QRCodeStyling({
        data: "https://google.com",
        width: 200,
        height: 200
    });

    qrCode.append(document.getElementById("canvas"));
    
    document.getElementById("dl").addEventListener("click", () => {
        browserUtils.download(qrCode, { extension: "svg" });
    });
    </script>
  </body>
</html>
```

Resulting svg file content:
```
<?xml version="1.0" standalone="no"?>
<?xml version="1.0" standalone="no"?>
<svg xmlns="http://www.w3.org/2000/svg" width="250" height="250" viewBox="0 0 250 250">[...]</svg>
```
![qr (3)](https://github.com/Liquid-JS/qr-code-styling/assets/1088043/80e74ce6-fa02-4df3-a5c9-63dc956b91e5)

**Comment by HitkoDev** [comment]
I see, there's been some leftover code in the browser utils. It's fixed in [3.0.2](https://github.com/Liquid-JS/qr-code-styling/releases/tag/v3.0.2)

**Comment by rplc** [comment]
Thanks for the quick fix. Works great.
